### DISCUSSION TITLE
Workflow Question: Audio Proxies

### DISCUSSION THREAD

**Discussion by moosefuel** [CommandPost/FCPCafe#252] [Final Cut Pro Questions] [answered]
I'll keep this short. I keep my media on a RAID by my desk and the project files on my MacBook (backed up, don't worry) which makes it really easy to make proxies and edit on the go. 

However, Final Cut Pro doesn't make audio proxies, and a lot of my interviews are dual-system (audio recorded separately) and so the audio doesn't come with me when I edit on the go. 

Anyone have a simple solution?

**Comment by d-mac-648** [comment, marked as answer]
Not really a short answer, but here's what I do...

You would create your proxy media and copy that folder to your "travel drive" (in your case, it would be the internal drive).

Since stills and audio aren't included, you create smart collections for those two types of media. Create a new temporary Library, select both smart collections in the sidebar and select all in the Browser and copy those to the new Library. If that new temp Library has media set to leave in place and you target a folder next to your proxy media, all of the still and audio will end up in that folder when you consolidate the audio and stills in the temp Library.

So, you'd have a proxy folder and an "Original Media" folder, containing the audio and stills. You can delete the temp Library. You'd need to move the original Library to the "travel" drive (in your case, you don't need to copy it since it's already on your internal drive).

Once you're "on the go", you can open your original Library, set the View menu to Proxy Preferred. You'd then select both smart collections for the audio and stills, select all, then relink. Now, you'll have your proxy a-v files, as well as the audio and stills to edit with.

When you return "home" and want to use your original/optimized media, you just change the View menu setting.

I've skipped a bunch of stuff, which shouldn't be hard to figure out (or you may not need to).

I learned all of this a few years ago in the Ripple Training "Media Management in FCP" tutorial...


Cheers.


P.S. - The way I've outlined above to get a copy of audio and stills to use "on the go" is easier than what is shown in this video (though this video is pretty good at explaining things, in lieu of their full tutorial): https://www.youtube.com/watch?v=kDCSFtvQ5SU

**Reply by moosefuel** [reply]
Great workflow! I’ll give it a try next time. Thank you! I guess Ripple Training is a good value.
